aio/karma.conf.js

```diff
@@ -1,6 +1,29 @@
 // Karma configuration file, see link for more information
 // https://karma-runner.github.io/1.0/config/configuration-file.html
 
+const os = require("os");
+
+if (os.platform() === "win32") {
+  /*
+  For some unknown reason, the symlinked copy of chromium under runfiles won't run under
+  karma on Windows. Instead, modify the CHROME_BIN env var to point to the chrome binary
+  under external/ in the execroot.
+
+  CHROME_BIN is set to the make var $(CHROMIUM) in aio/BUILD.bazel, which points to chrome
+  under runfiles and thus starts with '../'. Because we run architect with a chdir into aio,
+  the env var is given an additional leading '../'.
+
+  First, back out of
+      bazel-out/x64_windows-fastbuild/bin/aio/test.bat.runfiles/angular/aio
+  Then go into
+      external/
+  and then into
+      org_chromium_chromium_windows/chrome-win
+  to cancel out the leading ../../
+*/
```

Comment:
Nit: Inconsistent indentation:

```suggestion
    For some unknown reason, the symlinked copy of chromium under runfiles won't run under
    karma on Windows. Instead, modify the CHROME_BIN env var to point to the chrome binary
    under external/ in the execroot.

    CHROME_BIN is set to the make var $(CHROMIUM) in aio/BUILD.bazel, which points to chrome
    under runfiles and thus starts with '../'. Because we run architect with a chdir into aio,
    the env var is given an additional leading '../'.

    First, back out of
        bazel-out/x64_windows-fastbuild/bin/aio/test.bat.runfiles/angular/aio
    Then go into
        external/
    and then into
        org_chromium_chromium_windows/chrome-win
    to cancel out the leading ../../
  */
```